modules/serverless-logic-install-web-console.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * /serverless/install/install-serverless-operator.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-install-web-console_{context}"]
+= Installing the {ServerlessLogicOperatorName} from the web console
+
+You can install the {ServerlessLogicOperatorName} from OperatorHub by using the {ocp-product-title} web console. Installing this Operator enables you to install and use serverless workflows.
+
+.Prerequisites
+
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+* For {ocp-product-title}, your cluster has the Marketplace capability enabled or the Red Hat Operator catalog source configured manually.
+* You have logged in to the  web console.
+
+.Procedure
+
+. In the web console, navigate to the *Operators* -> *OperatorHub* page.
+
+. Scroll, or type the keyword *Serverless Logic* into the *Filter by keyword* box to find the {ServerlessLogicOperatorName}.
+
+. Review the information about the Operator and click *Install*.
+
+. On the *Install Operator* page:
```

Comment:
Lead-ins to lists should be complete sentences. Something along these lines maybe for this one?
```suggestion
. On the *Install Operator* page, ensure that the configuration uses the following settings:
```

Reply:
Understood and added